### PR TITLE
Add Acrechain to networks.json

### PR DESCRIPTION
**Pull request by SynergyNodes**

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -1330,6 +1330,22 @@
...
+    "network": "mainnet",
+    "multicall": "0x5B614BB4b2939dc4E5EC1399F16c7E78949764CB",
+    "rpc": [
+      "https://evm.acrescan.com"
+    ],
+    "explorer": {
+      "url": "https://acrescout.mindheartsoul.org"
+    },
+    "start": 1690100,
+    "logo": "ipfs://QmbA6CE3G81mm4eDEBUezwcq2qdqEh1zEToG3MbJHirxQt"
```
Comment (ChaituVR) [inline src/networks.json:1347]: Should pin this file using a provider like pinnata or web3storage etc. right now it is not resolving

File: src/networks.json [hunk 2]
```diff
@@ -1330,6 +1330,22 @@
     "start": 1683285,
     "logo": "ipfs://QmVh5XRi7xN4UNw7SNKpEhjF1TsPXJnRyNkB4EuMVTVCB6"
   },
+  "9052": {
+    "key": "9052",
+    "name": "Acrechain Mainnet",
+    "shortName": "Acrechain",
+    "chainId": 9052,
+    "network": "mainnet",
+    "multicall": "0x5B614BB4b2939dc4E5EC1399F16c7E78949764CB",
```
Comment (ChaituVR) [inline src/networks.json:1339]: Code here https://acrescout.mindheartsoul.org/address/0x5b614bb4b2939dc4e5ec1399f16c7e78949764cb/contracts#address-tabs should be verified and published

File: src/networks.json [hunk 3]
```diff
@@ -1330,6 +1330,22 @@
...
+    "rpc": [
+      "https://evm.acrescan.com"
+    ],
+    "explorer": {
+      "url": "https://acrescout.mindheartsoul.org"
+    },
+    "start": 1690100,
```
Comment (ChaituVR) [inline src/networks.json:1346]:
```suggestion
    "start": 471283,
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 1 above)
Comment (SynergyNodes) [reply src/networks.json:1347]: Can you clarify that you are talking about the logo ipfs file

Comment (SynergyNodes) [reply src/networks.json:1347]: > Should pin this file using a provider like pinnata or web3storage etc. right now it is not resolving

Can you clarify that you are talking about the logo ipfs file

Comment (ChaituVR) [reply src/networks.json:1347]: yep. Try going here https://ipfs.io/ipfs/QmbA6CE3G81mm4eDEBUezwcq2qdqEh1zEToG3MbJHirxQt should pin the file to pinning service else it won't resolve

File: src/networks.json (hunk 3 above)
Comment (SynergyNodes) [reply src/networks.json]: Hello,

1. We created new Multicall contract and code is verified.
2. We have pinned the LOGO (IPFS)
3. Changed the start block to the block number where the new Multicall was created.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**